prov/efa/src/efa_av.c

```diff
@@ -680,7 +691,7 @@ int efa_av_insert(struct fid_av *av_fid, const void *addr,
 	for (i = 0; i < count; i++) {
 		addr_i = (struct efa_ep_addr *) ((uint8_t *)addr + i * EFA_EP_ADDR_LEN);
 
-		ret = efa_av_insert_one(av, addr_i, &fi_addr_res, flags, context);
+		ret = efa_av_insert_one(av, addr_i, &fi_addr_res, flags, context, !!(av->shm_rdm_av));
```

Comment:
`av->shm_rdm_av` is checked inside `efa_conn_rdm_init`, passing `true` here should be enough.

Reply:
Yes, you are right, updated. Thx